README.md

```diff
@@ -28,148 +28,191 @@ inference.
     sudo apt-get install python3-pip python3-venv
     ```
 
-3. Create and activate virtual environment:
-    ```
-    python3 -m venv venv
-    source venv/bin/activate
-    ```
+3. Search for available scripts that create python virtual environments for different task types:
+   ```bash
+   find external/ -name init_venv.sh
+   ```
+   
+   Approximate output:
+   ```
+   external/mmdetection/init_venv.sh
+   external/mmsegmentation/init_venv.sh
+   external/deep-object-reid/init_venv.sh
+   ```
+   
+4. Let's create, activate Object Detection virtual environment and install `ote_cli`:
+   ```
+   ./external/mmdetection/init_venv.sh det_venv
+   source det_venv/bin/activate
+   pip3 install -e ote_cli/
+   ```
+   
+### OTE CLI tools
+1. Have a look at model templates available for this virtual environment:
+   ```
+   ote_find --root ./external/mmdetection/
```

Comment:
Just an idea. Make `ote` as an entrypoint to all tools. I.e. `ote_find` -> `ote find`.